### PR TITLE
Correct SignalK to Signal K

### PR DESCRIPTION
**Pull request by tedenda**
This PR is related to Issue #211.

### CODE REVIEW

File: examples/ultrasonic_level_sensor/README.md [hunk 1]
```diff
@@ -19,7 +19,7 @@
 
 ## Hardware
 
-The sensor has been implemented using an Adafruit Feather HUZZAH ESP8266 and SignalK/SensESP. It requires only a voltage divider for signal conditioning to reduce the 5V output pulse to 3 V and gets power directly from the HUZZAH (powered by 5v).
+The sensor has been implemented using an Adafruit Feather HUZZAH ESP8266 and Signal K/SensESP. It requires only a voltage divider for signal conditioning to reduce the 5V output pulse to 3 V and gets power directly from the HUZZAH (powered by 5v).
```
Comment (mairas) [inline examples/ultrasonic_level_sensor/README.md:22]: This one refers to the GitHub project name and should be left as is.

**Comment by mairas** [comment]
Looks good, except for one case that was referring to the GitHub project name.

For future reference: By writing "Fixes #nnn." in the PR description you can link an existing issue to the PR and have it automatically closed when the PR is merged. See: https://docs.github.com/en/free-pro-team@latest/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue

File: examples/ultrasonic_level_sensor/README.md (hunk 1 above)
Comment (tedenda) [reply examples/ultrasonic_level_sensor/README.md:22]: Ah, missed that!
Will change that and update PR.

**Comment by tedenda** [comment]
> Looks good, except for one case that was referring to the GitHub project name.
> 
> For future reference: By writing "Fixes #nnn." in the PR description you can link an existing issue to the PR and have it automatically closed when the PR is merged. See: https://docs.github.com/en/free-pro-team@latest/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue

Will check that out, thanks for teaching me new things!

File: examples/ultrasonic_level_sensor/README.md (hunk 1 above)
Comment (tedenda) [reply examples/ultrasonic_level_sensor/README.md]: Fixed!

**Comment by tedenda** [comment]
Okay now it should be golden!
Fixes #214

**Comment by mairas** [comment]
Great! Thanks a lot! Merging.

**Pull request merged**